packages/react-events/src/__tests__/Press-test.internal.js

```diff
@@ -477,6 +539,38 @@ describe('Event responder: Press', () => {
       expect(onLongPress).not.toBeCalled();
     });
 
+    it('is called but does not bubble', () => {
```

Comment:
Let's perform all the bubbling tests in the "nested responders" block

Reply:
I was going to remove the nested responders block, as I thought it would be best to be about type of event. :/ I don't mind though.

Reply:
Sure. But it seems we could test the bubbling/not-bubbling for all events in a single test.

Reply:
I'll move in a follow up tomorrow, will need some re-organizing of the test file.